config-linux.md

```diff
@@ -45,6 +45,17 @@ For Linux the mapping is
 
 capabilities is an array of Linux process capabilities. Valid values are the string after `CAP_` for capabilities defined in http://linux.die.net/man/7/capabilities
 
+## Linux Sysctl
+
+```
+   "sysctl": [
+        "net.ipv4.ip_forward": "1",
+        "net.core.somaxconn": "256"
+   ],
+```
+
+sysctl modifies kernel behavior at runtime by writing to files under /proc/.
```

Comment:
Perhaps just "sysctl allows kernel parameters to be modified at runtime" - the bit about /proc seems unnecessary (and nit, /proc/sys/)